src/parsers/power_system_table_data.jl

```diff
@@ -534,18 +534,19 @@ function load_csv_parser!(sys::System, data::PowerSystemTableData)
             throw(DataFormatError("Did not find bus index in Load data $(ps_bus.name)"))
         end
 
-
-        load = PowerLoad(
-            name = ps_bus.name,
-            available = true,
-            bus = ps_bus,
-            model = ConstantPower::LoadModel,
-            activepower = active_power,
-            reactivepower = reactive_power,
-            maxactivepower = max_active_power,
-            maxreactivepower = max_reactive_power,
-        )
-        add_component!(sys, load)
+        if (max_active_power != 0.0) | (max_reactive_power != 0.0)
```

Comment:
```suggestion
        if (max_active_power != 0.0) || (max_reactive_power != 0.0)
```

Reply:
Still correct here but you should really use a logical `or` instead of a bitwise `or`.